data/example_realization_config_w_bmi_c__linux.json

```diff
@@ -46,27 +46,61 @@
         "cat-27": {
             "formulations": [
                 {
-                    "name": "bmi_c",
+		    "name": "bmi_multi",
```

Comment:
Oooh, tricky... this file exists I think to give an example of a `bmi_c` formulation... but essentially CFE can't now run by itself with existing forcings.... so it has to exist in _multi ... can we make an example using exclusively `libpetbmi` instead? @hellkite500 ?

Reply:
Talked to the group this morning and we will probably just file a TODO to move these examples to the test modules, e.g. `extern/test_bmi_c`.